keanu-python/tests/test_coal_mining.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import keanu as kn
+import numpy as np
+
+def test_coalmining():
+    FILE = "data/coal-mining-disaster-data.csv"
+    data = pd.read_csv(FILE, names=["year", "count"]).set_index("year")
+    start_year, end_year = (data.index.min(), data.index.max())
+
+    with kn.Model() as m:
```

Comment:
I'm not personally a huge fan either but I think in Python it's so ubiquitous it would be considered incorrect to *not* use them - particularly with context managers etc.  The one exception is if you have a lot of them, but if our expected usage is that you only have one model "open" at a time (and not for hundreds of LoC) I think the current code is fine.

I've certainly had patches rejected for "unnecessarily long variable names"...

Reply:
😭